### DISCUSSION TITLE
How can I decrease the height of the header within the default theme?

### DISCUSSION THREAD

**Discussion by snuup** [vuejs/vitepress#4398] [Q&A] [answered]
I analyzed the css on the elements and read the docs but do not understand what makes the height of the header.

**Comment by brc-dd** [comment]
Which header? Can you show screenshot?

**Comment by snuup** [comment]
![image](https://github.com/user-attachments/assets/dad4410c-77f6-4108-bb6f-24fb8c06db1e)

**Reply by snuup** [reply, marked as answer]
made some progress, setting --vp-nav-height
